package.json

```diff
@@ -8,13 +8,14 @@
     "@turf/line-intersect": "^6.0.2",
     "@turf/points-within-polygon": "^5.1.5",
     "@types/node": "^13.11.1",
+    "jest": "^26.6.3",
```

Comment:
Strong thoughts on whether this should go here in root or down in example-project?

Reply:
It seems with using npm, we should install in the example package, there is no notion of a root package.json.  yarn workspaces supports this though, which is what I was used to

Reply:
moved